packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -17,7 +17,7 @@
                 method="post"
                 @submit.prevent="onFormSubmit">
                 <form-field
-                    :value="consumer.firstName"
+                    v-model="consumer.firstName"
```

Comment:
These changes were done because the inputs were not passing validations at any point, so they were broken. @kevinrodrigues will probably take a look at this in detail next week, and if not, we'll check it in the new year 😄

Reply:
What do you mean not "passing validations"?

Reply:
Basically, you'd put a valid value in any of the fields, and you'd get the red message under the field saying it was an invalid value.

Reply:
I changed the `:v-model` to `:value` because you should not bind a vuex store state model to a bi-direction binder, it will complain because it will try and update the state directly rather than going via a mutation.